src/pages/login.tsx

```diff
@@ -0,0 +1,49 @@
+import type { NextPage } from 'next';
+import Header from '../components/header'
+import Link from 'next/link'
+
+const Login: NextPage = () => {
```

Comment:
These fields and buttons should be tested. Just make sure that the `sign in` button doesn't work without an email and password entered. Could also check the API call it makes.

Reply:
Added the required HTML attribute to the input fields so the forms can't be submitted with a required field that's empty.